src/commands/timelinecommands.cpp

```diff
@@ -492,18 +492,18 @@ void MoveClipCommand::redo()
         }
     }
     // First, remove each clip.
+    int i = 0;
     for (auto &clip : m_selection) {
         if (!m_redo) {
             // On the initial pass, remove clips while recording info about them.
             QScopedPointer<Mlt::ClipInfo> info(m_model.findClipByUuid(MLT.uuid(clip), trackIndex, clipIndex));
             if (info && info->producer && info->producer->is_valid() && info->cut) {
-                info->producer->set(kNewTrackIndexProperty, qBound(0, trackIndex + m_trackDelta,
```

Comment:
The following constants seem to be unused after this change:
* kNewTrackIndexProperty
* kClipIndexProperty
* kShotcutInProperty
* kShotcutOutProperty

So we could remove them from shotcut_mlt_properties.h

Reply:
Yeah, They can be removed. 
I thought that it'd be better to do it in another commit because strictly speaking these deletions contribute nothing to fixing the current bug.

Reply:
It is fine as a separate commit, but I would like that commit to  be part of this PR so that we don't forget.

Reply:
OK. I don't mind deletions being part of this pull request.
It's going to be better to squash all the commits, though. I don't want "Restore kTrackIndexProperty kPlaylistStartProperty" a part of the commit history.